index.html

```diff
@@ -4,7 +4,7 @@
         <title>Dashboard</title>
         <meta charset="utf-8"/>
         <script src="https://kit.fontawesome.com/e2ff8c17ea.js" crossorigin="anonymous"></script>
-        <link rel="stylesheet" href="./dashboard.sass"/>
+        <link rel="stylesheet" href="./mediaphone.sass"/>
```

Comment:
you should include both sass files or @include one in the another. These styles are mandatory if you want to support multiple resolutions